LorentzVector.hh

```diff
@@ -57,6 +57,38 @@ namespace cola {
             return *this;
         }
 
+        // borrowed from CLHEP. bx by and bz are projections of beta
+        LorentzVectorImpl& boost(Type bx, Type by, Type bz) {
+            Type b2 = bx*bx + by*by + bz*bz;
+            if (b2 >= 1)
+                throw std::runtime_error("Boost faster than speed of light.");
+            Type ggamma = 1.0 / std::sqrt(1.0 - b2);
+            Type bp = bx*x + by*y + bz*z;
+            Type gamma2 = b2 > 0 ? (ggamma - 1.0)/b2 : 0.0;
```

Comment:
what is gamma2?

Reply:
A common factor in matrix of general Lorentz transformation

Reply:
Okay